### ISSUE TITLE
kytos Dockerfile

### ISSUE THREAD

**Issue by viniarck** [kytos/kytos#584]
Guys, two questions:

1 - Could you guys share kytos Dockerfile that is available on docker hub? I'm interested in deploying kytos in production in a container. I noticed there are several images available in the kytos account, probably the dev image has everything I need (mounting volumes for logging, for example). 
2 - Do you guys plan to provide official releases on Docker? I feel like the tryfirst, is just for quickly experimenting with kytos and/or testing. 

Basically, I'm setting a docker-compose file to deploy kytos with other container-based services that we plan to run at AmLight.

**Comment by beraldoleal** [comment]
Hi @viniciusarcanjo ...

1. Yes, we can share this. We can create a repo here named "dockers" with all our docker files.

2. Today, every time that we do a release we update our tryfirst. But for sure, because of 1 we need to create releases too.

If you have interest in maintaining such docker releases inside Kytos, please let us know. It will be a great help.

**Comment by viniarck** [comment]
Absolutely, @beraldoleal. Just let me know when this dockers repo is ready with the current Dockerfiles and I'll maintain. 

Thanks a lot!

**Comment by beraldoleal** [comment]
@macartur please move our docker repo to https://github.com/kytos/dockers
